TFNetwork.py

```diff
@@ -1967,6 +1972,57 @@ def copy_new_base(self, name=None, layer=None, network=None, reduce_func=None):
       norm_factor=self._norm_factor, only_on_eval=self._only_on_eval)
 
 
+class MetaLossHolder:
+  """
+  Combines multiple losses in a customizable way.
+  It can be eg. used to compute the average over multiple losses which may then control the learning rate.
+  Depending on the way the losses are combined, the result may not be normalized.
+  This class supports only a subset of the functions in LossHolder. It is not supposed to be used eg. to compute
+  the objective function.
+  """
+  def __init__(self, losses_dict, eval_string):
+    """
+    :param dict[str,TFNetworkLayer.Loss] losses_dict: Mapping from loss names to losses.
+    :param string eval_string: Formula to combine the relevant losses. Will be eval'ed.
```

Comment:
`string` is the wrong type. It is called `str`. (Please always check in PyCharm that the types are correctly inferred.)